openrtb_ext/imp_adhese.go

```diff
@@ -0,0 +1,82 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+type ExtImpAdhese struct {
+	Account  string          `json:"account"`
+	Location string          `json:"location"`
+	Format   string          `json:"format"`
+	Keywords json.RawMessage `json:"targets,omitempty"`
+}
+
+type ExtAdhese struct {
```

Comment:
Do not rename the original fields here. Use exactly the names that you currently have in AdheseBid. 
Then, you'll be able to marshal this object directly from the response.

From then on, I believe you should be just one step away of completely getting rid of AdheseOpenRtbBid.